docs2/site/docs/migrations/migration5.md

```diff
@@ -430,3 +449,33 @@ string s = writer.ToString();
 `SDLPrinter` is a highly optimized visitor for asynchronous non-blocking SDL output
 into provided `TextWriter`. In the majority of cases it does not allocate memory in
 the managed heap at all.
+
+### 21. `AutoRegisteringObjectGraphType` changes
+
+The protected method `GetRegisteredProperties` has been renamed to `GetRegisteredMembers`
+and now supports properties, methods and fields, although fields are not included
+with the default implementation. Override the method in a derived class to include fields.
+
+New for v5, methods are included by default.  To revert to v4 behavior, which does not
```

Comment:
> New for v5, methods are included by default.

Are you sure?

Reply:
Yes

Reply:
Methods were supported by the schema builder, but not by `AutoRegisteringObjectGraphType`.